package.json

```diff
@@ -138,6 +138,7 @@
     "typescript": "^4.9.5"
   },
   "dependencies": {
+    "faces-intellisense": "file:",
```

Comment:
@FlipWarthog i wonder how this will affect the plugin build?  Because this is a `dependency` it will try and be compiled into the plugin won't this be a circular dependency?